core/storage/wal.rs

```diff
@@ -75,11 +93,23 @@ pub struct WalFileShared {
     max_frame: u64,
     nbackfills: u64,
     // Maps pgno to frame id and offset in wal file
```

Comment:
is this comment correct? is it more like `Maps pgno to list of frame indexes in WAL that affect that page` or something...